source/v4/includes/_registry.md

```diff
@@ -126,3 +166,53 @@ Field | Type | Description
 ### Response
 
 Empty on success.
+
+## Getting The Entry URL
+
+```javascript--browser
+import { SkynetClient, genKeyPairFromSeed } from "skynet-js";
+
+const client = new SkynetClient();
+const { publicKey } = genKeyPairFromSeed("this seed should be fairly long for security");
+
+const dataKey = "foo";
+
+function getEntryExample() {
+  try {
+    const url = client.registry.getEntryUrl(publicKey, dataKey);
+  } catch (error) {
+    console.log(error);
+  }
+}
```

Comment:
Any reason to wrap this in a function with throw-catch ? I think the example should be as simple as possible like

```suggestion
const url = client.registry.getEntryUrl(publicKey, dataKey);
```